### PR TITLE
Refactoring components

### PR DESCRIPTION
**Pull request by kaylawoodbury**
### User Story
```
As a development team, 
In order to have readable code
I would like to have cleaner and smaller refactored components
```
## Changes proposed in this pull request:
* Extracted code from large screen files into sub components (multiple smaller files) for better readability
* Renamed files to be more clear as to their purpose
* Update Complete Trip button to only be visible when there is an active trip
## What I have learned working on this feature:
* Refactoring allows you to really examine the readability of your code and determine if there is a better way to write something. 
* Once time has passed and more has been added to the code base, it can be tricky to make sure you don't break everything when refactoring. Would probably have served us better to refactor as we developed the code initially instead of bulk refactoring later.

### CODE REVIEW

File: components/HomeScreenButtons.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                payload: { updateProfileMessage: "" },
+              });
+              getProfileInformation(userId, dispatch);
+            }}
+          >
+            <Text id="profile-button" style={styles.buttonText}>
+              My Profile
+            </Text>
+          </TouchableHighlight>
+  );
```
Comment (emtalen) [inline components/HomeScreenButtons.js:147]: Im not sure if the formatting is off here :)

**Review by emtalen — APPROVED** [review]
Looks great!

**Review by pgaunitz — APPROVED** [review]
Good job!

File: components/HomeScreenButtons.js (hunk 1 above)
Comment (kaylawoodbury) [reply components/HomeScreenButtons.js:147]: Reviewed and corrected :)

**Pull request merged**